doc/source/Getting_started/Troubleshooting.rst

```diff
@@ -0,0 +1,168 @@
+this can be run directly from AEDT Script menu and the wheelhouse zip file can be passed as an argument.
+
+
+
+
+Run PyAEDT
+~~~~~~~~~~
+
+COM vs GRPC
+-----------
+Up until AEDT 2022R2, the method for connecting Python to AEDT-API used COM.
+COM is a technology which requires interfaces, classes, objects and methods to be registered in Windows Registry.
+All communication between Python and AEDT-API were translated through an intermediate layer by a
+third party module called pywin32. This module usage was limited to Windows OS only.
+PythonNET was added to make the connection to the AEDT-API.
+In 2022R2, a new technology was added to replace COM: GRPC. GRPC is a modern remote procedure calls framework
+to communicate with the API via remote calls.
+
+
+.. list-table:: GRPC Compatibility Table
+   :widths: 25 25 75 75
+   :header-rows: 1
+
+   * -
+     - <2022R2
+     - 2022R2
+     - >2022R2
+   * - AEDT
+     - Only PythonNET
+     - GRPC Beta
+     - GRPC Available
+   * - AEDT
+     - Only PythonNET
+     - PythonNET default.
+       GRPC: pyaedt.settings.use_grpc_api = True
+     - GRPC default
+       PythonNET: pyaedt.settings.use_grpc_api = False
+
+Above options are available only in windows. Linux uses GRPC in any version as it is the only way to connect
+to AEDT API.
+
+
+How to check that Electronics Desktop API is working and correctly configured
+-----------------------------------------------------------------------------
+To start Electronics desktop server in gRPC mode use the following syntax:
+
+.. code:: python
+
+   path\to\AnsysEM\v231\Win64\ansysedt.exe -grpcsrv 50001   # Windows
+   path\to\AnsysEM\v231\Lin64\ansysedt -grpcsrv 50352       # Linux
+
+The number on which the server is running is the port that AEDT should use to listen and receive
+any command from PyAEDT. This allows to have multiple AEDT sessions running on the same machine
+and listening on the same port.
+
+Check that AEDT GRPC API can run
+--------------------------------
+Native Electronics Desktop API command can be used to launch Electronics Desktop from command line.
+This can be done even without PyAEDT to check that everything is setup correctly and all environment
+variables have been correctly setup.
+
+.. code:: python
+
+    import sys
+    sys.path.append(r"ANSYSEM_ROOT231\PythonFiles\DesktopPlugin")
+    import ScriptEnv
+    print(dir())
+    ScriptEnv.Initialize("", False, "", 50051)
+    print(dir())
+
+
+
+Failures in connecting to GRPC API
+----------------------------------
+On Linux, it may happens that PyAEDT fails to initialize a new session of Electronics Desktop
+or to connect to an existing one.
+This may be due to:
+ - Firewall
+ - Proxy
+ - Permissions
+ - License
+ - Scheduler used to launch AEDT like LSF
+
+In case of issues with proxy, you may try the following environment variable:
+
+.. code:: python
+
+    export no_proxy=localhost,127.0.0.1
+
+Run your PyAEDT script. If it still fails, then try:
```

Comment:
```suggestion
If running your PyAEDT script still fails, then try adding this:
```
A noun of some sort should follow "this". For example, "... try adding this line of code:" However, I'm not sure if that is technically correct. I'd appreciate your adding the appropriate noun.